### PR TITLE
Open Beta v0.21.0 Release

### PR DESCRIPTION
**Pull request by gary-kim**
Signed-off-by: Gary Kim <gary@garykim.dev>

### CODE REVIEW

**Comment by insertcustomname** [comment]
What exactly is the git flag you need to use for merging releases?

**Comment by gary-kim** [comment]
The instructions for making a release are here.
https://github.com/gary-kim/saspes/blob/322d9dac0d3a05eaf98773b1eecd15df9c271a96/CONTRIBUTING.md#make-a-new-release
I'm about to merge this in myself but could you do the next release?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "saspes",
-  "version": "0.20.0",
+  "version": "0.21.0",
   "version_name": "Open Beta 0.20.0",
```
Comment (gary-kim) [inline package.json:4]: Whoops, forgot to update the version name. Updated it for the release. PR incoming to fix that in the repo.

**Comment by insertcustomname** [comment]
How long does chrome take to auto update extensions?

**Comment by gary-kim** [comment]
The update is still stuck in "pending review". It should be available as soon as that is done.

**Comment by gary-kim** [comment]
Turns out Google rejected the update because I forgot to write explanations for each of the permissions we request. I've added that and resubmitted for review.

**Pull request merged**